Headlines/src/Controllers/ProfileViewController.swift

```diff
@@ -112,44 +107,37 @@ class ProfileViewController: UITableViewController, TabbedViewController {
         Answers.logCustomEvent(withName: "profile_appear", customAttributes: nil)
     }
     
-    // MARK: UITableViewDataSource
-    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return reactions.count
-    }
-    
-    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        guard let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath)
-            as? ReactionTableViewCell else {
-                return UITableViewCell()
-        }
-        
-        let r = reactions[indexPath.row]
-        guard let n = r.news else {
-            return cell
-        }
-        
-        cell.emojiLabel.text = r.reaction
-        cell.reactionLabel.text = n.title
-        cell.reactionImageView.sd_setImage(with: n.imageUrl)
-        
-        return cell
+    // MARK: UICollectionViewDelegate
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+//        trackOpenNews(n)
+//        openURL(n.url)
     }
     
-    // MARK: UITableViewDelegate
-    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        let r = reactions[indexPath.row]
-        
-        guard let n = r.news else {
-            return
+    func collectionView(_ collectionView: UICollectionView,
+                        layout collectionViewLayout: UICollectionViewLayout,
+                        sizeForItemAt indexPath: IndexPath) -> CGSize {
+
+        switch indexPath.section {
+        case 1:
+            let xInset = collectionView.layoutMargins.left
+            let yInset = collectionView.layoutMargins.top
+            let cellWidth = collectionView.bounds.insetBy(dx: xInset, dy: yInset).width
+            return CGSize(width: cellWidth, height: 60)
+
+        default:
+            guard let cell = profileDataSource.collectionView(collectionView, cellForItemAt: indexPath) as? LabelCollectionViewCell else {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/20270ea9d2349c8304e134bc47abf4f93c7a667a/Headlines/src/Controllers/ProfileViewController.swift#L128"></span>Line should be 120 characters or less: currently 138 characters</td>
    </tr>
  </tbody>
</table>